README.md

```diff
@@ -1,58 +1,75 @@
 # hcdiag
+`hcdiag` simplifies debugging HashiCorp products by automating shared and product-specific diagnostics data collection on individual nodes. The output of these commands is bundled up into a tar.gz bundle in the destination directory `hcdiag` is run 
```

Comment:
I think maybe the wording wasn't quite what was intended; perhaps this suggestion is closer?
```suggestion
`hcdiag` simplifies debugging HashiCorp products by automating shared and product-specific diagnostics data collection on individual nodes. The output of these commands is bundled up into a tar.gz bundle in the destination directory where `hcdiag` is run.
```